src/components/forms/pool_actions/MigrateForm/components/PoolsInfo/PoolsInfo.vue

```diff
@@ -188,7 +188,7 @@ const isUnstakedMigrationEnabled = computed(() => {
     </div>
     <PoolInfoBreakdown :pool="fromPool" :poolTokenInfo="fromPoolTokenInfo" />
     <div class="flex justify-center my-4 dark:text-gray-50">
-      <ArrowDownIcon class="w-5 h-5 dark:text-secondary" />
+      <ArrowDownIcon class="w-5 h-5 text-secondary" />
```

Comment:
Why are we getting rid of the dark: modifier here?

Reply:
We were having this [issue](https://github.com/tailwindlabs/tailwindcss/issues/2807) because text-secondary in our tailwind.css:

```
  .text-secondary {
    @apply text-gray-600 dark:text-gray-400;
  }
```

Already contains a dark modifier, so using dark:text-secondary is not possible. 

However, removing dark was not enough (my fault) as we explicitly wanted a dark style in both light and dark modes, so I finally replaced  the old `dark:text-secondary` by `dark:text-gray-400`.

Reply:
Note: `text-secondary` is not a tailwind style. It is a custom global css rule that I introduced so that we could have a dark grey on light backgrounds and a lighter grey on dark backgrounds to have better contrast accessibility ratios.

This means that `dark:text-secondary` should not work. For secondary gray text, you can just use `text-secondary`. Note: This can only be used in html (not in @apply rules within scoped css).

Reply:
Ok, it's fine to remove the dark modifier then.